server/ngwmn/templates/provider.html

```diff
@@ -0,0 +1,34 @@
+{% extends 'base.html' %}
+
+
+{% set page_title = agency_metadata.agency_nm + ' (' + agency_metadata.agency_cd + ')' %}
+
+{% block content %}
+    {{ provider_content | safe }}
+    <div class="usa-accordion">
+        <h3 class="usa-accordion-heading">
+            <button class="usa-accordion-button" aria-expanded="false" aria-controls="site-selection-container">
+                Site Selection and Classification
+            </button>
+        </h3>
+        <div id="site-selection-container" class="usa-accordion-content">{{ site_selection | safe }}</div>
+        <h3 class="usa-accordion-heading">
+            <button class="usa-accordion-button" aria-expanded="false" aria-controls="data-collection-container">
+                Data Collection Techniques
+            </button>
+        </h3>
+        <div id="data-collection-container" class="usa-accordion-content">{{ data_collection | safe }}</div>
+        <h3 class="usa-accordion-heading">
+            <button class="usa-accordion-button" aria-expanded="false" aria-controls="data-management-container">
+                Data Management
+            </button>
+        </h3>
+        <div id="data-management-container" class="usa-accordion-content">{{ data_management | safe }}</div>
+        <h3 class="usa-accordion-heading">
+            <button class="usa-accordion-button" aria-expanded="false" aria-controls="other-agency-info-container">
+                Other Agency Information
+            </button>
+        </h3>
+        <div id="other-agency-info-container" class="usa-accordion-content">{{ other_agency_info | safe }}</div>
+    </div>
+{% endblock %}
```

Comment:
I'm mixed with our use of accordions - it makes it harder to skim the page and jump around visually. What would you think about putting all these in a right-side column on desktop sizes without accordions?

Reply:
In this case it follows what was done previously on the old provider pages except there tabs where used. I would be okay with eliminating the accordian and just having the subsections right in the page. I would rather do that then start to put things in columns especially since we have less control over the content on these.

Reply:
Maybe these pages shouldn't be full width since they are more texty (not a word!) then the site page.

Reply:
It must not be the case with 1.0, but USWDS 2.0 has a default max-width set on `p` tags due to some kind of UX research concerning readability.

Reply:
we are overriding that?

Reply:
In the meantime, I'm going to remove the accordions because I don't like them either.

Reply:
Oh, this is 2.0-alpha. Hmm, I don't know.... maybe they didn't introduce it until one of the later betas.